stix_shifter_modules/aws_athena/stix_transmission/query_connector.py

```diff
@@ -35,10 +37,14 @@ def create_query_connection(self, query):
                 query = json.loads(query)
             query_service_type = list(query.keys())[0]
             config_details = service_types[query_service_type]
+            if all([True if config not in self.connection.keys() else False for config in config_details]):
+                random_id = ''.join([random.choice(string.ascii_letters + string.digits) for _ in range(30)])
```

Comment:
why do you need to generate id like this? can't you just generate a random UUID?

Reply:
@mdazam1942 The code change for generating UUID using the python package is done and pushed to PR branch.